boxoffice/models/order.py

```diff
@@ -62,6 +62,7 @@ class Order(BaseMixin, db.Model):
     buyer_fullname = db.Column(db.Unicode(80), nullable=False)
     buyer_phone = db.Column(db.Unicode(16), nullable=False)
 
+    # TODO: Rename to receipt_no
     invoice_no = db.Column(db.Integer, nullable=True)
```

Comment:
Why are we not using Coaster's existing `BaseScopedIdMixin` then? It includes a safety harness (server-side increments) learnt from failures in production.